app/src/main/java/com/kickstarter/ui/activities/PaymentMethodsSettingsActivity.kt

```diff
@@ -19,21 +20,33 @@ import com.kickstarter.viewmodels.PaymentMethodsViewModel
 import com.stripe.android.paymentsheet.PaymentSheet
 import com.stripe.android.paymentsheet.PaymentSheetResult
 import com.stripe.android.paymentsheet.model.PaymentOption
-import rx.android.schedulers.AndroidSchedulers
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
 
-@RequiresActivityViewModel(PaymentMethodsViewModel.ViewModel::class)
-class PaymentMethodsSettingsActivity : BaseActivity<PaymentMethodsViewModel.ViewModel>() {
+class PaymentMethodsSettingsActivity : AppCompatActivity() {
 
     private lateinit var adapter: PaymentMethodsAdapter
     private var showDeleteCardDialog: AlertDialog? = null
 
     private lateinit var binding: ActivitySettingsPaymentMethodsBinding
     private lateinit var flowController: PaymentSheet.FlowController
 
+    private lateinit var viewModelFactory: PaymentMethodsViewModel.Factory
```

Comment:
Notice we generate now the ViewModel using a Factory and android architecture component `ViewModelProvider.Factory`